internal/commands/app/delete_inputs.go

```diff
@@ -0,0 +1,85 @@
+package app
+
+import (
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+var (
+	flagApps      = "apps"
+	flagAppsUsage = "the Realm app names or ids to manage"
+)
+
+type deleteInputs struct {
+	Apps    []string
+	Project string
+}
+
+func (inputs *deleteInputs) resolveApps(ui terminal.UI, client realm.Client) ([]realm.App, error) {
+	apps, err := client.FindApps(realm.AppFilter{GroupID: inputs.Project})
+	if err != nil {
+		return nil, err
+	}
+
+	if len(inputs.Apps) > 0 && len(apps) == 0 {
+		return nil, cli.ErrAppNotFound{}
+	}
+
+	if len(apps) == 0 {
+		return apps, nil
+	}
+
+	if len(inputs.Apps) > 0 {
+		appsByName := map[string]realm.App{}
+		for _, app := range apps {
+			appsByName[app.Name] = app
+		}
+
+		missingApps := make([]string, 0)
+		appsFiltered := make([]realm.App, 0, len(inputs.Apps))
+		for _, inputApp := range inputs.Apps {
+			app, ok := appsByName[inputApp]
+			if !ok {
+				missingApps = append(missingApps, inputApp)
+				continue
+			}
+			appsFiltered = append(appsFiltered, app)
+		}
+
+		if len(missingApps) > 0 {
+			ui.Print(terminal.NewWarningLog(
+				"unable to delete certain apps because they were not found: %s",
+				strings.Join(missingApps, ", "),
+			))
+		}
+
+		if len(appsFiltered) > 0 {
+			return appsFiltered, nil
```

Comment:
question: would we want to just `return` here always, regardless if the length is 0 or not?  that way they get the warning that none of their apps were found, but we don't then also prompt them to pick from a list of apps

Reply:
Hmm that is a good point. Thinking about this, I agree with your point of not prompting in this scenario. I will change this to `if len(inputs.Apps) > 0` so we can return appsFiltered if the user specified app inputs